files/ical.js/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "ical.js",
+  "repo": "mozilla-comm/ical.js",
+  "files": {
+    "include": ["build/ical.js", "build/ical.min.js", "build/ical.min.js.map"]
```

Comment:
There is no min.js here https://github.com/mozilla-comm/ical.js/tree/v1.1.2/build